src/test/java/atlastests/SearchDefs.java

```diff
@@ -120,6 +120,7 @@ public static boolean isFileDownloaded(String downloadPath, String fileName) {
 
     @Then("^file download$")
     public void fileDownload() throws Exception {
+	Thread.sleep(5000); // wait for file to download
```

Comment:
it is a bad practice. if it's possible, please, rewrite this part with waitUntil structure or with Selenide.sleep(5000)(but waitUntil is much better).